### ISSUE TITLE
Evaluation metrics - Regression / Generative models

### ISSUE THREAD

**Issue by csudre** [Project-MONAI/MetricsReloaded#40]
**Is your feature request related to a problem? Please describe.**
A list of the metrics to be implemented to be applied in the case of a regression/generative task with possible links for implementation / reference

- Mean absolute error [https://pytorch.org/docs/master/generated/torch.nn.L1Loss.html#torch.nn.L1Loss](url)
- Mean squared error [https://pytorch.org/docs/master/generated/torch.nn.MSELoss.html#torch.nn.MSELoss](url)
- Smooth L1 Loss [https://pytorch.org/docs/master/generated/torch.nn.SmoothL1Loss.html#torch.nn.SmoothL1Loss](url)
- Root MSE [](url)
- Peak Signal to Noise Ratio [https://github.com/tensorflow/tensorflow/blob/v2.2.0/tensorflow/python/ops/image_ops_impl.py#L3358](url)
- Structural Similarity Index Measure [https://github.com/tensorflow/tensorflow/blob/v2.2.0/tensorflow/python/ops/image_ops_impl.py#L3569](url)
- Multi-scale SSIM [https://github.com/tensorflow/tensorflow/blob/v2.2.0/tensorflow/python/ops/image_ops_impl.py#L3648](url
- Negative Guassian Log Likelihood [https://pytorch.org/docs/stable/nn.html#torch.nn.NLLLoss](url)
- Maximum Mean Discrepancy [https://github.com/ZongxianLee/MMD_Loss.Pytorch/blob/master/mmd_loss.py#L5](url)
- Inception Score [https://github.com/sbarratt/inception-score-pytorch](url)
- Frechet Inception Distance [https://github.com/xuqiantong/GAN-Metrics](url)

**Additional context**
This comes from the work on metrics by task type done by the Metrics Task Force

**Comment by csudre** [comment]
This issue is not currently covered by the Metrics Reloaded guidelines - this is for future work
